gui/index.html

```diff
@@ -97,9 +97,10 @@
 		<!-- ADDING CUSTOM BLOCKS -->
 	</xml>
 
-	<!-- basic imports -->
+	<!-- basic imports (NEED TO BE LOCALIZED) -->
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
 	<script src="http://pixijs.download/release/pixi.min.js"></script>
+	<script src="https://d3js.org/d3.v5.min.js"></script>
```

Comment:
Please add a TODO to change this to be a local import.

Reply:
In fact it may be a good idea to make an issue alongside this. We should have everything installed via `npm`.